components/infra-proxy-service/server/cookbooks.go

```diff
@@ -39,13 +39,18 @@ func (s *Server) GetCookbookVersions(ctx context.Context, req *request.CookbookV
 		return nil, status.Errorf(codes.InvalidArgument, "invalid org ID: %s", err.Error())
 	}
 
-	cookbook, err := c.client.Cookbooks.GetAvailableVersions(req.Name, "")
+	res, err := c.client.Cookbooks.GetAvailableVersions(req.Name, "")
 	if err != nil {
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
+	cookbook, success := res[req.Name]
```

Comment:
the response is returning cookbook name as map key and versions as `CookbookVersion` struct.

So, checking the cookbook name `success` for sanity purposes only.